infra.yaml

```diff
@@ -452,6 +452,7 @@ resources:
             str_replace:
               params:
                 $WC_NOTIFY: { get_attr: ['wait_handle', 'curl_cli'] }
+                $container_quota: ''
```

Comment:
I wonder if it wouldn't be better to pass container_quota as another docker_set_storage_device function's parameter?
something like:
docker_set_storage_device $DOCKER_VOLUME_ID $CONTAINER_QUOTA in *-boot scripts

This would allow us to keep the common functions library as standalone as possible.